core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -464,7 +464,7 @@ oppia.directive('conversationSkin', [
               // If the exploration is embedded, use the exploration language
               // as site language. If the exploration language is not supported
               // as site language, English is used as default.
-              var langCodes = $window.GLOBALS.SUPPORTED_SITE_LANGUAGES.map(
+              var langCodes = $window.constants.SUPPORTED_SITE_LANGUAGES.map(
```

Comment:
Manually tested and debugged both. Works fine either way. Anyway I've removed $window now for conciseness.